### ISSUE TITLE
make docs-all fails

### ISSUE THREAD

**Issue by seenickcode** [Byron/google-apis-rs#281]
```
➜  google-apis-rs git:(main) make docs-all
.api.deps:3645: warning: overriding commands for target `etc/api/dns/v1/crates/2.0.5-api+20210319'
.api.deps:3614: warning: ignoring old commands for target `etc/api/dns/v1/crates/2.0.5-api+20210319'
.cli.deps:3762: warning: overriding commands for target `etc/api/dns/v1/crates/2.0.5-cli+20210319'
.cli.deps:3730: warning: ignoring old commands for target `etc/api/dns/v1/crates/2.0.5-cli+20210319'
python3 -m virtualenv -p python3 .pyenv-Darwin
created virtual environment CPython3.8.2.final.0-64 in 124ms
  creator CPython3macOsFramework(dest=/Users/seenickcode/code/google-apis-rs/.pyenv-Darwin, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/Users/seenickcode/Library/Application Support/virtualenv)
    added seed packages: Jinja2==2.11.3, Mako==1.1.4, Markdown==3.3.4, MarkupSafe==1.1.1, PyYAML==5.4.1, attrs==20.3.0, certifi==2020.12.5, chardet==4.0.0, click==7.1.2, codecov==2.1.11, coverage==5.5, future==0.18.2, ghp_import==1.1.0, idna==2.10, importlib_resources==5.1.2, iniconfig==1.1.1, joblib==1.0.1, livereload==2.6.3, lunr==0.5.8, mkdocs==1.1.2, nltk==3.6.2, packaging==20.9, pip==20.3.1, pip==21.1, pluggy==0.13.1, py==1.10.0, pyparsing==2.4.7, pytest==6.2.3, pytest_cov==2.11.1, python_dateutil==2.8.1, regex==2021.4.4, requests==2.25.1, setuptools==51.0.0, six==1.15.0, toml==0.10.2, tornado==6.1, tqdm==4.60.0, urllib3==1.26.4, wheel==0.36.1
  activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator
.pyenv-Darwin/bin/python -m pip install -r requirements.txt
Requirement already satisfied: mako==1.1.4 in ./.pyenv-Darwin/lib/python3.8/site-packages (from -r requirements.txt (line 1)) (1.1.4)
Requirement already satisfied: pyyaml in ./.pyenv-Darwin/lib/python3.8/site-packages (from -r requirements.txt (line 2)) (5.4.1)
Requirement already satisfied: mkdocs==1.1.2 in ./.pyenv-Darwin/lib/python3.8/site-packages (from -r requirements.txt (line 3)) (1.1.2)
Requirement already satisfied: pytest in ./.pyenv-Darwin/lib/python3.8/site-packages (from -r requirements.txt (line 4)) (6.2.3)
Requirement already satisfied: pytest-cov in ./.pyenv-Darwin/lib/python3.8/site-packages (from -r requirements.txt (line 5)) (2.11.1)
Requirement already satisfied: importlib_resources in ./.pyenv-Darwin/lib/python3.8/site-packages (from -r requirements.txt (line 6)) (5.1.2)
Requirement already satisfied: codecov in ./.pyenv-Darwin/lib/python3.8/site-packages (from -r requirements.txt (line 7)) (2.1.11)
Requirement already satisfied: ghp-import in ./.pyenv-Darwin/lib/python3.8/site-packages (from -r requirements.txt (line 8)) (1.1.0)
Requirement already satisfied: coverage in ./.pyenv-Darwin/lib/python3.8/site-packages (from codecov->-r requirements.txt (line 7)) (5.5)
Requirement already satisfied: requests>=2.7.9 in ./.pyenv-Darwin/lib/python3.8/site-packages (from codecov->-r requirements.txt (line 7)) (2.25.1)
Requirement already satisfied: python-dateutil>=2.8.1 in ./.pyenv-Darwin/lib/python3.8/site-packages (from ghp-import->-r requirements.txt (line 8)) (2.8.1)
Requirement already satisfied: MarkupSafe>=0.9.2 in ./.pyenv-Darwin/lib/python3.8/site-packages (from mako==1.1.4->-r requirements.txt (line 1)) (1.1.1)
Requirement already satisfied: tornado>=5.0 in ./.pyenv-Darwin/lib/python3.8/site-packages (from mkdocs==1.1.2->-r requirements.txt (line 3)) (6.1)
Requirement already satisfied: lunr[languages]==0.5.8 in ./.pyenv-Darwin/lib/python3.8/site-packages (from mkdocs==1.1.2->-r requirements.txt (line 3)) (0.5.8)
Requirement already satisfied: livereload>=2.5.1 in ./.pyenv-Darwin/lib/python3.8/site-packages (from mkdocs==1.1.2->-r requirements.txt (line 3)) (2.6.3)
Requirement already satisfied: Jinja2>=2.10.1 in ./.pyenv-Darwin/lib/python3.8/site-packages (from mkdocs==1.1.2->-r requirements.txt (line 3)) (2.11.3)
Requirement already satisfied: click>=3.3 in ./.pyenv-Darwin/lib/python3.8/site-packages (from mkdocs==1.1.2->-r requirements.txt (line 3)) (7.1.2)
Requirement already satisfied: pyyaml in ./.pyenv-Darwin/lib/python3.8/site-packages (from -r requirements.txt (line 2)) (5.4.1)
Requirement already satisfied: Markdown>=3.2.1 in ./.pyenv-Darwin/lib/python3.8/site-packages (from mkdocs==1.1.2->-r requirements.txt (line 3)) (3.3.4)
Requirement already satisfied: pluggy<1.0.0a1,>=0.12 in ./.pyenv-Darwin/lib/python3.8/site-packages (from pytest->-r requirements.txt (line 4)) (0.13.1)
Requirement already satisfied: packaging in ./.pyenv-Darwin/lib/python3.8/site-packages (from pytest->-r requirements.txt (line 4)) (20.9)
Requirement already satisfied: iniconfig in ./.pyenv-Darwin/lib/python3.8/site-packages (from pytest->-r requirements.txt (line 4)) (1.1.1)
Requirement already satisfied: py>=1.8.2 in ./.pyenv-Darwin/lib/python3.8/site-packages (from pytest->-r requirements.txt (line 4)) (1.10.0)
Requirement already satisfied: toml in ./.pyenv-Darwin/lib/python3.8/site-packages (from pytest->-r requirements.txt (line 4)) (0.10.2)
Requirement already satisfied: attrs>=19.2.0 in ./.pyenv-Darwin/lib/python3.8/site-packages (from pytest->-r requirements.txt (line 4)) (20.3.0)
Requirement already satisfied: coverage in ./.pyenv-Darwin/lib/python3.8/site-packages (from codecov->-r requirements.txt (line 7)) (5.5)
Requirement already satisfied: pytest in ./.pyenv-Darwin/lib/python3.8/site-packages (from -r requirements.txt (line 4)) (6.2.3)
Requirement already satisfied: MarkupSafe>=0.9.2 in ./.pyenv-Darwin/lib/python3.8/site-packages (from mako==1.1.4->-r requirements.txt (line 1)) (1.1.1)
Requirement already satisfied: tornado>=5.0 in ./.pyenv-Darwin/lib/python3.8/site-packages (from mkdocs==1.1.2->-r requirements.txt (line 3)) (6.1)
Requirement already satisfied: six in ./.pyenv-Darwin/lib/python3.8/site-packages (from livereload>=2.5.1->mkdocs==1.1.2->-r requirements.txt (line 3)) (1.15.0)
Requirement already satisfied: future>=0.16.0 in ./.pyenv-Darwin/lib/python3.8/site-packages (from lunr[languages]==0.5.8->mkdocs==1.1.2->-r requirements.txt (line 3)) (0.18.2)
Requirement already satisfied: six in ./.pyenv-Darwin/lib/python3.8/site-packages (from livereload>=2.5.1->mkdocs==1.1.2->-r requirements.txt (line 3)) (1.15.0)
Requirement already satisfied: nltk>=3.2.5 in ./.pyenv-Darwin/lib/python3.8/site-packages (from lunr[languages]==0.5.8->mkdocs==1.1.2->-r requirements.txt (line 3)) (3.6.2)
Requirement already satisfied: future>=0.16.0 in ./.pyenv-Darwin/lib/python3.8/site-packages (from lunr[languages]==0.5.8->mkdocs==1.1.2->-r requirements.txt (line 3)) (0.18.2)
Requirement already satisfied: six in ./.pyenv-Darwin/lib/python3.8/site-packages (from livereload>=2.5.1->mkdocs==1.1.2->-r requirements.txt (line 3)) (1.15.0)
Requirement already satisfied: joblib in ./.pyenv-Darwin/lib/python3.8/site-packages (from nltk>=3.2.5->lunr[languages]==0.5.8->mkdocs==1.1.2->-r requirements.txt (line 3)) (1.0.1)
Requirement already satisfied: click>=3.3 in ./.pyenv-Darwin/lib/python3.8/site-packages (from mkdocs==1.1.2->-r requirements.txt (line 3)) (7.1.2)
Requirement already satisfied: tqdm in ./.pyenv-Darwin/lib/python3.8/site-packages (from nltk>=3.2.5->lunr[languages]==0.5.8->mkdocs==1.1.2->-r requirements.txt (line 3)) (4.60.0)
Requirement already satisfied: regex in ./.pyenv-Darwin/lib/python3.8/site-packages (from nltk>=3.2.5->lunr[languages]==0.5.8->mkdocs==1.1.2->-r requirements.txt (line 3)) (2021.4.4)
Requirement already satisfied: pyparsing>=2.0.2 in ./.pyenv-Darwin/lib/python3.8/site-packages (from packaging->pytest->-r requirements.txt (line 4)) (2.4.7)
Requirement already satisfied: six in ./.pyenv-Darwin/lib/python3.8/site-packages (from livereload>=2.5.1->mkdocs==1.1.2->-r requirements.txt (line 3)) (1.15.0)
Requirement already satisfied: chardet<5,>=3.0.2 in ./.pyenv-Darwin/lib/python3.8/site-packages (from requests>=2.7.9->codecov->-r requirements.txt (line 7)) (4.0.0)
Requirement already satisfied: certifi>=2017.4.17 in ./.pyenv-Darwin/lib/python3.8/site-packages (from requests>=2.7.9->codecov->-r requirements.txt (line 7)) (2020.12.5)
Requirement already satisfied: idna<3,>=2.5 in ./.pyenv-Darwin/lib/python3.8/site-packages (from requests>=2.7.9->codecov->-r requirements.txt (line 7)) (2.10)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in ./.pyenv-Darwin/lib/python3.8/site-packages (from requests>=2.7.9->codecov->-r requirements.txt (line 7)) (1.26.4)
mkdocs gen/abusiveexperiencereport1-cli/build_html/index.html
ERROR   -  Config value: 'pages'. Error: Invalid pages config. {<class 'list'>} {<class 'dict'>, <class 'str'>} 

Aborted with 1 Configuration Errors!
make: *** [gen/abusiveexperiencereport1-cli/build_html/index.html] Error 1
```

I'm using 

```
Python 2.7.16 (default, Dec 21 2020, 23:00:36) 
[GCC Apple LLVM 12.0.0 (clang-1200.0.30.4) [+internal-os, ptrauth-isa=sign+stri on darwin
```

**Comment by Byron** [comment]
Thanks for letting me know!

This issue should be fixed now and CI will now keep it that way :).

**Comment by seenickcode** [comment]
Thank you!

On April 25, 2021, Sean Pianka ***@***.***> wrote:
> Closed #281 <https://github.com/Byron/google-apis-rs/issues/281>.
>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Byron/google-apis-rs/issues/281#event-4641966200>,
or unsubscribe <https://github.com/notifications/unsubscribe-
auth/AAASQOJVNQK7DAZEKPXUSQ3TKS723ANCNFSM43R3E5BQ>.
